src/shared.ts

```diff
@@ -24,3 +35,43 @@ export const fontConfig = {
     ios: defaultFontConfig,
     android: defaultFontConfig,
 };
+
+export type ThemeOverrides = {
+    avatar: {
+        background: string;
+    };
+
+    bottomNavigation: {
+        inactive: string;
+    };
+    button: {
+        contained: {
+            text: {
+                disabled: string;
+            };
+        };
+    };
+    chip: {
+        outlined: {
+            background: string;
+        };
+        flat: {
+            background: string;
+        };
+    };
+    snackbar: {
+        accent: string;
+    };
+    toggleButtonGroup: {
+        checked: {
+            background: string;
+        };
+    };
+};
+
+export type ThemeOpacity = {
+    disabled: number;
+    divider: number;
+    disabledBackground: number;
+    actionOpacity: number;
+};
```

Comment:
You can add an export statement to the index.ts file for these, so that users can import them to use in their global type override.